packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -90,12 +93,20 @@ export class SerializedStateManager {
 			"readBlob" | "getSnapshotTree" | "getSnapshot" | "getVersions"
 		>,
 		private readonly _offlineLoadEnabled: boolean,
+		private readonly containerEvent: EventEmitter,
```

Comment:
```suggestion
		private readonly containerEvent: IEventProvider<{(event:"saved",()=>void)}>,
```

Reply:
Typescript doesn't like `event: "saved"` for some reason